src/pages/Designer/Designer.js

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import {
+  Grommet,
+  Box,
+  Heading,
+  Text,
+  Stack,
+  Card,
+  CardBody,
+  Paragraph,
+  Image,
+} from 'grommet';
+import {
+  ClearOption,
+  Configure,
+  Camera,
+  CloudUpload,
+  Selection,
+  BarChart,
+} from 'grommet-icons';
+import { grommetToolsTheme } from '../../theme.js';
+import AppHeader from '../../components/AppHeader.js';
+import IconCard from '../../components/IconCard.js';
+
+function Designer() {
+  return (
+    <Grommet theme={grommetToolsTheme}>
```

Comment:
Shimi mentioned on my PR that we shouldn't have `<Grommet>` in each individual page and it should instead just be in the container for the application

Reply:
Fixed in the most recent push